org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -48,6 +48,7 @@ public void testCoverageResult() {
 		// 3. Try/Catch Block Without Exception Thrown
 		assertLine("noExceptionTryCatch.beforeBlock", ICounter.FULLY_COVERED);
 		assertLine("noExceptionTryCatch.tryBlock", ICounter.FULLY_COVERED);
+		assertLine("noExceptionTryCatch.catch", ICounter.NOT_COVERED);
```

Comment:
This assertion fails with the Eclipse compiler (probably that's why I removed it at all):

    Status in line 75: 		} catch (StubException e) { // $line-noExceptionTryCatch.catch$ expected:<[NOT]_COVERED> but was:<[PARTLY]_COVERED>